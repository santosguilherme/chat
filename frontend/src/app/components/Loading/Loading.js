import React from 'react';
import CircularProgress from '@material-ui/core/CircularProgress';

import {LoadingContent} from './styles/Container';

function Loading() {
  return (
    <LoadingContent>
      <CircularProgress/>
    </LoadingContent>
  );
}

export default Loading;
