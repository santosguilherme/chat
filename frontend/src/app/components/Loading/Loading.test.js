import React from 'react';

import { customRender } from 'commons/utils/test/reactTestingLibrary';

import Loading from './Loading';

describe('Loading', () => {
  test('renders correctly', () => {
    const { container } = customRender(<Loading />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
