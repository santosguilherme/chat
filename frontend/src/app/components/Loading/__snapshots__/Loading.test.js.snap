// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
.c0 {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  class="c0"
>
  <div
    class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style="width: 40px; height: 40px;"
  >
    <svg
      class="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx="44"
        cy="44"
        fill="none"
        r="20.2"
        stroke-width="3.6"
      />
    </svg>
  </div>
</div>
`;
